eq-author/public/index.html

```diff
@@ -8,6 +8,16 @@
     <meta http-equiv="expires" content="0" />
     <meta http-equiv="expires" content="Tue, 01 Jan 1980 1:00:00 GMT" />
     <meta http-equiv="pragma" content="no-cache" />
+    <script>
+      var isChrome =
+        !!window.chrome &&
+        (!!window.chrome.webstore || !!window.chrome.runtime);
+      var isElectron = !!process.versions.electron;
```

Comment:
this will blow up as `process` isn't a global in most browsers.
```suggestion
      var isElectron = process && process.versions && process.versions.electron;
```

Reply:
I have added a proper function for it now.